### PR TITLE
fixed step number for dark mode

### LINKED ISSUE #1214: text not visible in dark mode
#### Current Behavior
The text 01 & 02 are not clearly visible
image

here too:
image

#### Expected Behavior
The 01 & 02 should be green /black  in color or any other color

Please assign me this issue

### PR DESCRIPTION
**Pull request by ranjanmangla1**
**Description**

This PR fixes #1214

Now the step numbers are clearly visible:
image

present condition on meshery.io site: 
image

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -67,17 +67,9 @@ GEM
...
       google-protobuf (~> 3.21)
       rake (>= 10.0.0)
-    sass-embedded (1.57.1-arm64-darwin)
-      google-protobuf (~> 3.21)
-    sass-embedded (1.57.1-x64-mingw32)
-      google-protobuf (~> 3.21)
-    sass-embedded (1.57.1-x86_64-darwin)
-      google-protobuf (~> 3.21)
-    sass-embedded (1.57.1-x86_64-linux)
-      google-protobuf (~> 3.21)
```
Comment (Chadha93) [inline Gemfile.lock:80]: This can be reverted. @ranjanmangla1

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@ranjanmangla1, the changes look good, revert the dependency changes

**Comment by Ghat0tkach** [comment]
Hii @ranjanmangla1 
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by thisiskaransgit** [comment]
PR abandoned...

**Pull request closed without merging**